repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/cta-card.ftl

```diff
@@ -0,0 +1,30 @@
+<#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"] >
+
+<#macro ctaCard ctaCard>
+    <#if ctaCard??>
+        <#if ctaCard.ctaCardContentBlock.ctaLink.document??>
+            <#assign link>
+                <@hst.link hippobean=ctaCard.ctaCardContentBlock.ctaLink.document/>
+            </#assign>
+            <#assign openInNewWindow=false/>
+        <#else>
+            <#assign link = "${ctaCard.ctaCardContentBlock.ctaLink.url}">
+            <#assign openInNewWindow=true/>
+        </#if>
+        <#if ctaCard.ctaCardContentBlock.ctaLink.text?has_content && link?has_content>
+            <div class="nhsuk-grid-column-one-third">
+            <div class="nhsuk-card nhsuk-card--clickable">
+                <div class="nhsuk-card__content">
+                    <h2 class="nhsuk-card__heading">

```

Comment:
Could we use `<h3>` for title (to be inline with other right hand side cards/blocks) ?

Reply:
Yes, commited on d5a8bce.